devops/code-coverage.yml

```diff
@@ -0,0 +1,68 @@
+# code-coverage.yml
+trigger: none
+pr: none
+
+schedules:
+  - cron: "30 5 * * *" # Time is UTC
+    displayName: Nightly Code Coverage Build
+    branches:
+      include:
+        - master
+        - dev
+    always: true
+
+pool:
+  vmImage: "ubuntu-latest"
+
+variables:
+  artifactFeed: interpret-extensions-feed
```

Comment:
maybe we need to create a new feed, can be done in a later PR

Reply:
true